Sorting/ItemSorter.cs

```diff
@@ -98,7 +100,7 @@ public static List<Item> Aggregate(AggregateContext context, CancellationToken t
 
 				List<Item> aggregate = new();
 
-				foreach (Item item in context.items.OrderBy(i => i.type).ThenBy(i => i.prefix))
+				foreach (Item item in context.items.OrderBy(i => i.type).ThenBy(i => i.prefix).ToList())
```

Comment:
`.ToList()` should be avoided to prevent iterating over the collection twice

Reply:
there was a reason for me trying it, I encountered a strange Collection was modified error. Will have to investigate further, have removed it for now though